roles/servicetelemetry/tasks/component_qdr.yml

```diff
@@ -38,7 +181,7 @@
             linkCapacity: 25000
         interRouterListeners:
           - authenticatePeer: true
-            expose: true
+            expose: false
```

Comment:
I wanted to get this opened up and note early in this process that I want to change the behaviour here to simplify the certificate creation.

In STF 1.4 the `interRouterListeners` would be created and exposed as Route, meaning something like `default-interconnect-55671.apps.[external_host_URL]` would be exposed, and then references as a `dnsName` in the creation of the `default-interconnect-inter-router-credentials` certificate.

This complicates the values to be found in order to make sure the `dnsNames` list is updated with the external route URL. Because we don't have a use case where we expect to build inter-router mesh networks outside of the current namespace, I don't believe setting `expose: true` is appropriate, and in theory this could be considered a hardening tactic (although I did this to reduce the needed logic to find and populate the appropriate inter-router Route creation dynamically).

Reply:
I endorse this change. I'm pretty sure this route was for the Cloudforms-compatible OSP-controller-mesh-mode feature, so that the OSP-side mesh would connect to STF in inter-router mode and not require additional edge nodes. AFAIK this is no longer supported.